lib/platform/git/storage.ts

```diff
@@ -152,17 +167,17 @@ class Storage {
       return git.status();
     }
 
-    async function createBranch(branchName, sha) {
+    async function createBranch(branchName: string, sha: string) {
       logger.debug(`createBranch(${branchName})`);
       await git.reset('hard');
       await git.raw(['clean', '-fd']);
       await git.checkout(['-B', branchName, sha]);
-      await git.push(['origin', branchName, '--force']);
+      await git.push('origin', branchName, { '--force': true });
```

Comment:
This change is unrelated to TS, right?

Reply:
yes, should do the same, the array overload is missing in types, so i have to use the options overload